### DISCUSSION TITLE
Issue Forms: add a short summary / required fields status

### DISCUSSION THREAD

**Discussion by Zerotask** [community/community#4349] [Projects and Issues] [answered]
The new issue forms looks really great. Just a little addition to it:
at the bottom, you could add a short summary: `1 / 3 required fields are filled out`

This summary could change, depending on the status:
if all required fields are filled out, it's green with a check.

If there are missing required fields, it could be red or orange and you can click it. Then it will jump to the first / next missing required field.

This way it could improve the UX even more, because people have a status and see directly why the can't click on the "Submit new issue" button.

![image](https://user-images.githubusercontent.com/20150243/123525686-d03bfa80-d6d2-11eb-838a-21f31d1f6786.png)

**Comment by Timmmy-nlb** [comment, marked as answer]
Good News: "required" is now working in Internal Repos!
https://github.com/orgs/community/discussions/148713#discussioncomment-12226996
